project/plugins.sbt

```diff
@@ -1,7 +1,5 @@
 addSbtPlugin("edu.gemini"         % "sbt-gsp"                  % "0.2.5")
 addSbtPlugin("com.geirsson"       % "sbt-ci-release"           % "1.5.3")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.2.0")
```

Comment:
I expect we'd use e.g. `User` in the client (but that will move to core). Would we have logic here that needs to go to the client.
Perhaps to decode the tokens?

Reply:
[Last time we discussed this](https://gemini-software.slack.com/archives/GJQDX830D/p1600181975468300) it sounded like you wouldn't need to look at the cookie on the client, but if it turns out you do then we can add a little client lib to help with it.